api/api-npm/__test__/data/transitive-deps/project/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "Project",
+    "version": "0.1.0",
+    "dependencies": {
+        "A": "file:../A-0.1.0.tgz",
```

Comment:
I would suggest to have one more layer of transitive dependenices just to be more sure it works